meteor/lib/Settings.ts

```diff
@@ -1,25 +1,22 @@
+import { Meteor } from 'meteor/meteor'
 import * as _ from 'underscore'
 
-const defaultSettings = require('./defaultSettings.json')
-let localSettings = {}
-try {
-	localSettings = require('../settings.json')
-} catch (e) {
-	localSettings = {}
+export interface ISettings {
+	frameRate: number,
+	defaultToCollapsedSegments: boolean,
+	autoExpandCurrentNextSegment: boolean
 }
 
-let localSettingsEnv = {}
-function screamSnakeCase (input: string) {
-	return input.replace(/([A-Z])/g, '_$1').toUpperCase()
-}
+export let Settings: ISettings
 
-try {
-	_.keys(defaultSettings).forEach((setting) => {
-		const envName = screamSnakeCase(setting)
-		if (process.env[envName] !== undefined) localSettingsEnv[setting] = process.env[envName]
-	})
-} catch (e) {
-	localSettingsEnv = {}
+const DEFAULT_SETTINGS: ISettings = {
+	"frameRate": 25,
+	"defaultToCollapsedSegments": false,
+	"autoExpandCurrentNextSegment": false
 }
 
-export const Settings = _.extend({}, defaultSettings, localSettings, localSettingsEnv)
+Settings = _.extend({}, DEFAULT_SETTINGS)
```

Comment:
Just a minor note, this isn't type safe as _.extend returns an `any` type. What you probably want to do instead:

`Settings = _.clone(DEFAULT_SETTINGS)`

Reply:
Good tip! I'll change that to `_.clone`.